src/global/tracking/tracking.cc

```diff
@@ -17,7 +17,7 @@
 #include "TrackerMeasurementFromHits_factory.h"
 #include "extensions/jana/JChainMultifactoryGeneratorT.h"
 #include "extensions/jana/JOmniFactoryGeneratorT.h"
-#include "factories/tracking/TrackerHitCollector_factory.h"
+#include "factories/digi/CollectionCollector_factory.h"
```

Comment:
Might be useful to create a new type for this, `meta` or `generic`. It is as little a digi factory as a tracking factory.
```suggestion
#include "factories/meta/CollectionCollector_factory.h"
```